vm/devices/vmbus/vmbus_server/src/lib.rs

```diff
@@ -231,6 +232,8 @@ impl EventPort for ChannelEvent {
 pub struct SavedState {
     #[mesh(1)]
     server: channels::SavedState,
+    #[mesh(2)]
+    lost_synic_bug_fixed: Option<bool>,
```

Comment:
Missing fields in saved states will just get the default "zero" value, which is false for bool. So you can just make this `bool`.